microprofile.ls/com.redhat.microprofile.ls/src/test/java/com/redhat/microprofile/ls/JavaTextDocumentSnippetRegistryTest.java

```diff
@@ -40,18 +40,25 @@ public void javaSnippets() {
 		Optional<Snippet> metricSnippet = findByPrefix("@Metric", registry);

```

Comment:
I think we should split the only test in 3 tests:

 * haveJavaSnippets
 * metricsSnippets
 * faultTolerantSnippets

Reply:
Updated to split the test into 3 tests. Plus added 1 more for the mp open api snippets.